win64svc/src/questdb.c

```diff
@@ -110,6 +110,7 @@ void buildJavaArgs(CONFIG *config) {
         strcat(args, classpath);
         strcat(args, "\"");
     }
+    strcat(args, " -Dbin");
```

Comment:
it feels "bin" will be rapidly reduced to wtf is this in couple of moth time. It could be a good idea to introduce system property, that would tell Java that it runs inside a container. Provided Java doesn't have a way of knowing this already.